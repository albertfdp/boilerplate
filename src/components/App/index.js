import React from 'react'

import styles from './styles.css'

const App = () => (
  <div className={styles.container}>
    <h1>An opinionated boilerplate</h1>
  </div>
)

export default App
