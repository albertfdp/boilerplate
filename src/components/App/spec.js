import React from 'react'
import expect from 'test/expect'

import App from '.'

describe('App', () => {
  it('renders', () => {
    expect(<App />, 'to match snapshot')
  })

  it('renders with', () => {
    expect(
      <App />,
      'to have rendered',
      <div>foo</div>
    )
  })
})
