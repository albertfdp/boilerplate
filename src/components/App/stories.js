import React from 'react'
import { storiesOf } from '@storybook/react'

import App from './'

storiesOf('App', module).add('renders', () => <App />)
