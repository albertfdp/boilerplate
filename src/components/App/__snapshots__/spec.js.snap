// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<div
  className="container"
>
  <h1>
    An opinionated boilerplate
  </h1>
</div>
`;
